drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1715,6 +1728,27 @@ function DragAndDropBlock(runtime, element, configuration) {
                     state.grade = data.grade;
                     if (!data.correct) {
                         delete state.items[item_id];
+
+                        try{
+                            failure_audio.pause();
```

Comment:
define a function and pass that function player reference, move `try-catch` in that function and call that function with `failure_audio` & `success_audio`